gulpfile.js

```diff
@@ -65,6 +66,26 @@ var buildConfig = {
       vendor: {
         src: 'source/frontend/vendor/js/development/vendor.js',
         dest: 'build/public/vendor/js'
+      },
+      bundles: {
+        index: {
+          entries: 'source/frontend/js/index.js',
+          require: [],
+          output: 'index.js',
+          dest: 'build/public/js'
+        },
+        bubbleShooter: {
+          entries: 'source/frontend/js/games/bubble-shooter.js',
+          require: [],
+          output: 'bubble-shooter.js',
+          dest: 'build/public/js/games'
+        },
+        vendor: {
+          entries: 'source/frontend/vendor/js/development/vendor.js',
+          require: ['jquery', 'q', 'lodash'],
```

Comment:
This `require` line is what makes _jquery_, _q_, and _lodash_ accessible to other script files.